src/pudl/metadata/classes.py

```diff
@@ -1748,3 +1749,58 @@ def to_rst(self, top_dir: DirectoryPath, csv_subdir: DirectoryPath, rst_path: st
                 rendered = encoder.to_rst(
                     top_dir=top_dir, csv_subdir=csv_subdir, is_header=header)
                 f.write(rendered)
+
+
+class DatasetteMetadata(Base):
+    """
+    A collection of Data Sources and Resources for metadata export.
+
+    Used to create metadata YAML file to accompany Datasette.
+    """
+
+    datasource_dict: Dict[str, DataSource]
+    resource_package: Package = Package.from_resource_ids()
+    label_column_dict: Dict[str, str]
```

Comment:
I think the column labels are going to be pretty fixed and might get a bit complicated. If they can't be generated in an automatic / semi-automatic way, I think it might make sense for them to live in here (or one of the other metadata modules, maybe a `pudl.metadata.datasette`) rather than only being specified in the script that's being used to generate the metadata. Or at least the stored version could be the default value, and could be overridden by an input if the user wishes.

Reply:
Moved the column labels to the `DatasetteMetadata` class.